cli/pcluster/config/validators.py

```diff
@@ -783,6 +772,35 @@ def s3_bucket_validator(param_key, param_value, pcluster_config):
     return errors, warnings
 
 
+def fsx_lustre_auto_import_validator(param_key, param_value, pcluster_config):
+    errors = []
+    warnings = []
+
+    if param_value is not None and param_value != "NONE":
+        try:
+            fsx_section = pcluster_config.get_section("fsx")
+            fsx_import_path = fsx_section.get_param_value("import_path")
+            bucket = get_bucket_name_from_s3_url(fsx_import_path)
+            s3_bucket_region = boto3.client("s3").get_bucket_location(Bucket=bucket)["ResponseMetadata"][
+                "LocationConstraint"
+            ]
```

Comment:
I think this will break for buckets in us-east-1. Use `.get` to avoid that:  `s3_bucket_region = boto3.client("s3").get_bucket_location(Bucket=bucket).get("ResponseMetadata", {}).get("LocationConstraint")`

Reply:
Now that I look at the docs for `get_bucket_location`, it looks like "LocationConstraint" is a top-level field in the response dict, so this should be: `s3_bucket_region = boto3.client("s3").get_bucket_location(Bucket=bucket).get("LocationConstraint")`